docs/src/cfs_versions.dox

```diff
@@ -4,93 +4,64 @@
+  The REVISION number shall be incremented on changes that benefit from unique identification such as
+  bug fixes or major documentation updates.
+  The Revision number may also be updated if there are other changes contained within a release that make
+  it desirable for applications to distinguish one release from another.
+  WARNING: The revision number is set to the number 99 in development builds. To distinguish between development
+  builds refer to the BUILD_NUMBER and BUILD_BASELINE detailed in the section "Identifying Development Builds".
 
-  The Major, Minor, and Revision numbers are provided in this header file as part of the API
-  definition; this macro must expand to a simple integer value, so that it can be used in simple if directives by the macro preprocessor.
-
-  The Mission Version number shall be set to zero in all officially released packages, and is entirely reserved for the use of the mission. The Mission Version is provided as a simple macro defined in the cfe_platform_cfg.h header file.
-
-  <H2> Version Number Flexibility </H2>
-
-  The major number may increment when there is no breaking change to the API, if the changes are significant enough to
-  warrant the same level of attention as a breaking API change.
-
-  The minor number may increment when there have been no augmentations to the API, if changes are as significant as
-  additions to the public API.
-
-  The revision numbers may increment in implementations where no actual implementation-specific code has changed, if
-  there are other changes within the release with similar significance.
+  The Mission Version number is set to zero in all official releases, and is reserved for the mission use.
 
   <H2> How and Where Defined </H2>
 
-  The Major, Minor, and Revision components of the version are provided as simple macros defined in the cfe_version.h header file as part of the API definition; these macros must expand to simple integer values, so that they can be used in simple if directives by the macro preprocessor.
+  The version numbers are provided as simple macros defined in the cfe_version.h header file as part of the
+  API definition; these macros must expand to simple integer values, so that they can be used in simple if
+  directives by the macro preprocessor.
 
-  The Mission Version is provided as a simple macro defined in the cfe_platform_cfg.h header file. As delivered in official releases, these macros must expand to simple integer values, so that they can be used in simple macro preprocessor conditions, but delivered code should not prevent a mission from, for example, deciding that the Mission Version is actually a text string.
+  Note the Mission Version number is provided for missions to be able to identify unique changes they
```

Comment:
should we replace "mission" with "implementation" or something similar to make it more generic?

Reply:
Sounds like a significant change in terminology that would require a full document scrub if it's going to be consistent.  Maybe a new issue if you think it's worth it?  Would have code and document impacts.